resources/views/common/_navigationHeader.antlers.html

```diff
@@ -1,38 +1,60 @@
+  <ul class="-mx-2 flex list-none items-center w-full">
     {{ nav:header }}
       {{ if depth == 1 }}
-        <li class="group/parent relative px-4">
-          <a
-            href="{{ url }}"
-            class="relative mt-[6px] inline-block cursor-pointer py-1 font-flama-book text-lg tracking-wider transition-colors"
-          >
+        <li class="group/parent relative px-4 normal-nav">
+          <a href="{{ url }}"
+            class="relative mt-[6px] inline-block cursor-pointer py-1 font-flama-book text-lg tracking-wider transition-colors">
             <span class="relative z-10 whitespace-nowrap">{{ title }}</span>
             <span
-              class="absolute bottom-0 left-0 h-[3px] w-0 bg-minova-gold transition-all duration-300 ease-in-out group-hover/parent:w-full"
-            ></span>
+              class="absolute bottom-0 left-0 h-[3px] w-0 bg-minova-gold transition-all duration-300 ease-in-out group-hover/parent:w-full"></span>
           </a>
           {{ if children }}
             <ul
-              class="absolute left-0 top-full z-40 mt-0 hidden min-w-[200px] list-none bg-white/90 px-4 pb-4 pt-10 shadow-lg transition-all group-hover/parent:block"
-            >
+              class="absolute left-0 top-full z-40 mt-0 hidden min-w-[200px] list-none bg-white/90 px-4 pb-4 pt-10 shadow-lg transition-all group-hover/parent:block">
               {{ *recursive children* }}
             </ul>
           {{ /if }}
         </li>
       {{ elseif depth == 2 }}
-        <li class="px-2 py-2">
-          <a
-            href="{{ url }}"
-            class="group/child relative w-full pb-1 font-flama-book text-base tracking-wider transition-colors group-hover/child:text-minova-gold"
-          >
+        <li class="px-2 py-2 normal-nav">
+          <a href="{{ url }}"
+            class="group/child relative w-full pb-1 font-flama-book text-base tracking-wider transition-colors group-hover/child:text-minova-gold">
             <span class="relative z-10 whitespace-nowrap">{{ title }}</span>
             <span
-              class="absolute bottom-0 left-0 h-[3px] w-0 bg-minova-gold transition-all duration-300 ease-in-out group-hover/child:w-full"
-            ></span>
+              class="absolute bottom-0 left-0 h-[3px] w-0 bg-minova-gold transition-all duration-300 ease-in-out group-hover/child:w-full"></span>
           </a>
         </li>
       {{ /if }}
     {{ /nav:header }}
+    <li class="group/parent relative px-4 normal-nav search-button">
+      <button onclick="toggleSearch()"
+        class="relative inline-block cursor-pointer py-1 font-flama-book text-lg tracking-wider transition-colors">
+        <span class="relative z-10 whitespace-nowrap uppercase">Search</span>
+        <span
+          class="absolute bottom-0 left-0 h-[3px] w-0 bg-minova-gold transition-all duration-300 ease-in-out group-hover/parent:w-full"></span>
+      </button>
+    </li>
   </ul>
-</nav>
+
```

Comment:
![Screenshot 2024-11-08 at 16 16 32](https://github.com/user-attachments/assets/753a1c19-50b2-42ad-90bb-413e7ab1e305)

This doesn't quite look right, I think a few things need to happen to make this look nicer:

- Instead of a slide in, just have the element fade in
- Remove the Big button and instead just have a chevron in a circle
- See if you can change the header a bit to have this instead just fill the block where the current nav exists (so that the logo stays visible)... this will also mean the search bar doesn't go full screen, it will be inside the container